spec/ios_generate_strings_file_from_code_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+require 'tmpdir'
+
+describe Fastlane::Actions::IosGenerateStringsFileFromCode do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_generate_strings_file_from_code') }
+  let(:app_src_dir) { File.join(test_data_dir, 'sample-project', 'Sources') }
+  let(:pod_src_dir) { File.join(test_data_dir, 'sample-project', 'Pods') }
+
+  def test_genstrings(paths_to_scan:, quiet:, swiftui:, expected_dir_name:, expected_logs: nil)
+    Dir.mktmpdir('a8c-wpmrt-ios_generate_strings_file_from_code-') do |tmp_dir|
+      # Act
+      genstrings_logs = described_class.run(paths: paths_to_scan, quiet: quiet, swiftui: swiftui, output_dir: tmp_dir)
+
+      output_files = Dir[File.join(tmp_dir, '*.strings')]
+      expected_files = Dir[File.join(test_data_dir, expected_dir_name, '*.strings')]
+
+      # Assert: same list of generated files
+      expect(genstrings_logs).to eq(expected_logs) unless expected_logs.nil?
+      expect(output_files.map { |f| File.basename(f) }.sort).to eq(expected_files.map { |f| File.basename(f) }.sort)
```

Comment:
Move comment next to the appropriate line it is commenting:
```suggestion
      # Assert: stdout/stderr warning messages from genstrings
      expect(genstrings_logs).to eq(expected_logs) unless expected_logs.nil?
      # Assert: same list of generated files
      expect(output_files.map { |f| File.basename(f) }.sort).to eq(expected_files.map { |f| File.basename(f) }.sort)
```